.claude/commands/commit-push.md

```
@@ -0,0 +1,74 @@
+
+   ### The Seven Rules of a Great Commit Message
+   1. Separate subject from body with a blank line
+   2. Limit the subject line to 50 characters (72 is the hard limit)
+   3. Capitalize the subject line
+   4. Do not end the subject line with a period
+   5. Use the imperative mood - "Add feature" not "Added feature"
+   6. Wrap the body at 72 characters
+   7. Use the body to explain what and why, not how
+
+   ### Commit Message Format
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)